subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/LargeDependencyGraphPerformanceTest.groovy

```diff
@@ -57,7 +57,7 @@ class LargeDependencyGraphPerformanceTest extends AbstractCrossVersionPerformanc
             runner.args += '--parallel'
         }
         if (improvedPomSupport) {
-            runner.args += '-Porg.gradle.advancedpomsupport=true -PimprovedPomSupport=true'
+            runner.args += '-PimprovedPomSupport=true'
```

Comment:
I believe this change is safe because the baseline is now `4.8-20180405000006+0000`

Reply:
This is not entirely true. We have _historical builds_ which test multiple versions of Gradle, to compare the performance of builds for the same scenario. You can see an example here: https://builds.gradle.org/viewLog.html?buildId=12099904&buildTypeId=Gradle_Check_PerformanceHistoricalCoordinator&tab=report_project947_Performance

This means that it is possible that we run this test with 4.6, as far as I know. I thought this was what `minimalVersion` was for (do not try to run before 4.0), but when I look at this: https://builds.gradle.org/viewLog.html?buildId=12099904&buildTypeId=Gradle_Check_PerformanceHistoricalCoordinator&tab=report_project947_Performance

It seems we started with 4.6-rc-1, so I don't really understand if the historical build is broken, or if it's intended. Best ask @oehme .

If it is supposed to run with older versions of Gradle, it means you need to generate the correct argument based on the version to run.

Reply:
The historical build is working as expected, but the list of tested versions hasn't been updated in a long time. It was just `2.9,2.12,2.14.1,last`. I updated it to `2.14.1,3.5.1,4.0,last`.

Reply:
Given my understanding of the introduction of improved bom support, it appeared in 4.5 behind properties. In 4.6 it was moved to the `enableFeaturePreview` API. Given the list from @oehme it should be safe to do the change since the only impacted version would be 4.5 and that's not in the list. Does that make sense?

Reply:
You should not depend on that list. You should properly set the `minimumVersion` of your test.